### PR TITLE
Feature add methods to bridge dubbo-hessian2 and kitex-hessian

### PR DESCRIPTION
**Pull request by Lvnszn**
#### What type of PR is this?

add methods to bridge dubbo-hessian2 and kitex-hessian

#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.
- [ ] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: tests/kitex/kitex_gen/echo/api.go [hunk 1]
```diff
@@ -6,11 +8,7 @@ import (
 )
 
 type EchoRequest struct {
-	Int32 int32 `json:"int32"`
-}
-
-func (p *EchoRequest) JavaClassName() string {
```
Comment (felix021) [inline tests/kitex/kitex_gen/echo/api.go:12]: 这个 JavaClassName 方法被删掉，后面 register 里不是会报错么？

Comment (Lvnszn) [reply tests/kitex/kitex_gen/echo/api.go:12]: 我目前是放在在 k-api 里面的

File: tests/kitex/kitex_gen/echo/k-api.go [hunk 2]
```diff
@@ -1,519 +1,334 @@
...
-	_ hessian2.Message = (*TestServiceEchoInt8sResult)(nil)
-	_ hessian2.Message = (*TestServiceEchoArgs)(nil)
-	_ hessian2.Message = (*TestServiceEchoResult)(nil)
+	_ = thrift.TProtocol(nil)
 )
 
-type TestServiceEchoIntArgs struct {
-	Req int32 `json:"req"`
-}
+func (p *EchoRequest) Encode(e codec.Encoder) error {
```
Comment (felix021) [inline tests/kitex/kitex_gen/echo/k-api.go:20]: The `Encode` and `Decode` functions are not used?

File: tests/kitex/kitex_gen/echo/register.go [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+
+var objects = []interface{}{
+	&EchoRequest{},
+	&EchoResponse{},
+}
 
 func init() {
-	hessian.RegisterPOJO(&EchoRequest{})
-	hessian.RegisterPOJO(&EchoResponse{})
+	register(objects)
```
Comment (felix021) [inline tests/kitex/kitex_gen/echo/register.go:11]: 这里可以直接调用 hessian2.Register，不用 register 函数了

**Review by felix021 — APPROVED** [review]

File: tests/kitex/kitex_gen/echo/testservice/testservice.go [hunk 4]
```diff
@@ -18,27 +19,26 @@ func NewServiceInfo() *kitex.ServiceInfo {
...
-		"PackageName": "echo",
+		"PackageName":     "echo",
+		"ServiceFilePath": "tool/cmd/kitex/api.thrift",
 	}
 	svcInfo := &kitex.ServiceInfo{
 		ServiceName:     serviceName,
 		HandlerType:     handlerType,
 		Methods:         methods,
-		KiteXGenVersion: "v0.6.1",
+		PayloadCodec:    kitex.Thrift,
```
Comment (felix021) [inline tests/kitex/kitex_gen/echo/testservice/testservice.go:34]: Should be Hessian2 here?

**Review by jasondeng1997 — APPROVED** [review]

**Comment by jasondeng1997** [comment]
LGTM

**Pull request merged**